query/logicalplan/validate.go

```diff
@@ -167,17 +167,23 @@ func ValidateAggregation(plan *LogicalPlan) *PlanValidationError {
 }
 
 func ValidateAggregationExpr(plan *LogicalPlan) *ExprValidationError {
+	// TODO: Check that aggregation columns don't have the same result name
+
 	// check that the aggregation expression has the required structure
 	colFinder := newTypeFinder((*Column)(nil))
-	plan.Aggregation.AggExpr.Accept(&colFinder)
+	for _, expr := range plan.Aggregation.AggExprs {
+		expr.Accept(&colFinder)
+	}
 
 	aggFuncFinder := newTypeFinder((*AggregationFunction)(nil))
-	plan.Aggregation.AggExpr.Accept(&aggFuncFinder)
+	for _, expr := range plan.Aggregation.AggExprs {
+		expr.Accept(&aggFuncFinder)
+	}
 
 	if colFinder.result == nil || aggFuncFinder.result == nil {
 		return &ExprValidationError{
 			message: "aggregation expression is invalid. must contain AggregationFunction and Column",
-			expr:    plan.Aggregation.AggExpr,
+			expr:    plan.Aggregation.AggExprs[0], // TODO
```

Comment:
Do we intend to leave these TODO in here, if so can we add a comment as to why?

Reply:
This is one left over from my side. I want to fix. 
Additionally, I'll address the comments made here too.